paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt

```diff
@@ -71,6 +76,11 @@ internal class DefaultCustomerSheetLoader(
                 configuration = configuration,
                 elementsSessionWithMetadata = elementsSessionWithMetadata,
             ).getOrThrow()
+        }.onFailure {
+            errorReporter.report(
+                errorEvent = ErrorReporter.ErrorEvent.CUSTOMER_SHEET_LOAD_FAILURE,
+                stripeException = StripeException.create(it)
+            )
```

Comment:
I put this as a catch-all but I'm wondering if we should separate this out into other error types for `customer_sheet`.